File: Task1/main.ts
function main() {
    const num: number = Number(prompt("Enter a number:"));
    
    for (let i = 1; i <= num; i++) {
        let line = '';
        for (let j = 1; j <= i; j++) {
            const ch: string = String.fromCharCode(num - j + 'A'.charCodeAt(0));
            line += ch + ' ';
        }
        console.log(line.trim());
    }
}

main();

File: Task1/main.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let num: i32 = input.trim().parse().expect("Please enter a number");

    for i in 1..=num {
        for j in 1..=i {
            let ch = (num - j + 'A' as i32) as u8 as char;
            print!("{} ", ch);
        }
        println!();
    }
}